src/streamlit_extras/star_rating/__init__.py

```diff
@@ -0,0 +1,82 @@
+from htbuilder import span, styles
+from htbuilder.units import px
+from streamlit import streamlit as st
+from streamlit.components.v1 import html
+
+
+def star_rating(rating: float, color="#FFD700"):
```

Comment:
`rating` should be `int | float`, not just `float`

Reply:
`color` can be type-hinted as `str`:
```suggestion
def star_rating(rating: int | float, color: str = "#FFD700"):
```